### PR TITLE
Remove some instances of TopCoder logging

### PR DESCRIPTION
**Pull request by jasonaowen**
Some of the logging in our app is done through a TopCoder utility library, which we have checked in as a jar in our repo. Replace a few instances of this logging with standard Java logging, as used elsewhere in the app, to get us closer to no longer depending on the TopCoder utility library.

Issue #214 Remove TopCoder-specific jarfiles

### CODE REVIEW

File: psm-app/cms-business-process/src/main/java/gov/medicaid/verification/BackgroundStudyClient.java [hunk 1]
```diff
@@ -36,6 +33,9 @@
 import java.io.IOException;
 import java.io.StringReader;
 import java.io.StringWriter;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.INFO;
```
Comment (cecilia-donnelly) [inline psm-app/cms-business-process/src/main/java/gov/medicaid/verification/BackgroundStudyClient.java]: Why import this here and not in the other files?

File: psm-app/cms-business-process/src/main/java/gov/medicaid/services/impl/BusinessProcessServiceBean.java [hunk 2]
```diff
@@ -196,7 +200,7 @@ public long enroll(EnrollmentType enrollment) throws Exception {
                 try {
                     ksession.dispose();
                 } catch (Throwable t) {
-                    getLog().log(Level.ERROR, t, "Could not close session.");
+                    LOGGER.log(SEVERE, "Could not close session.", t);
```
Comment (cecilia-donnelly) [inline psm-app/cms-business-process/src/main/java/gov/medicaid/services/impl/BusinessProcessServiceBean.java]: Why `LOGGER.log(SEVERE` here where I've seen `LOGGER.severe(` elsewhere?

Comment (jasonaowen) [reply psm-app/cms-business-process/src/main/java/gov/medicaid/services/impl/BusinessProcessServiceBean.java]: In this case we have both a message and a `Throwable`, and there isn't a version of the `severe` function that includes a `Throwable`. Other calls to `severe` did not include a `Throwable`, only a `String`.

File: psm-app/cms-business-process/src/main/java/gov/medicaid/verification/BackgroundStudyClient.java (hunk 1 above)
Comment (jasonaowen) [reply psm-app/cms-business-process/src/main/java/gov/medicaid/verification/BackgroundStudyClient.java]: We need it for the expression `LOGGER.isLoggable(INFO)` on line 104; the contents of that `if` block are complicated/expensive enough that we don't want to calculate what to log unless we know for sure we're going to log it.

Comment (jasonaowen) [reply psm-app/cms-business-process/src/main/java/gov/medicaid/verification/BackgroundStudyClient.java]: Arguably it'd be better to use the Java 8 lambda/supplier logger call ([example](https://stackoverflow.com/a/21751955/25637)), but this class is likely to be deleted at some point in the not-too-distant future as it was built to deal with the legacy external data sources, so I didn't want to spend too much time on it.

**Pull request merged**